config/mongodbatlas.spc

```diff
@@ -0,0 +1,10 @@
+connection "mongodbatlas" {
+  plugin = "mongodbatlas"
+  
+  # Public and Private Key Pair with 'Organization Read Only' permission
+  # These can also be 'MONGODB_ATLAS_PUBLIC_KEY' and/or 'MONGODB_ATLAS_PRIVATE_KEY'
+  # Consult https://www.mongodb.com/docs/atlas/configure-api-access/#create-an-api-key-in-an-organization on how to generate API keys
+  # public_key = "public key here"
```

Comment:
For this config arg, and the others, can you please update the example text here to use more realistic, dummy values, as per https://steampipe.io/docs/develop/plugin-release-checklist#configuration-file? For examples, please see https://github.com/turbot/steampipe-plugin-salesforce/blob/main/config/salesforce.spc.

Reply:
Please remember to make the same updates in `docs/index.md` as well in the `Configuration` section